packages/utils/feed/queries.ts

```diff
@@ -96,9 +96,12 @@ export const getPostCategory = ({
   title,
   files,
   message,
+  gifs,
 }: NewPostFormValues): PostCategory => {
   let category: PostCategory;
-  if (files?.length) {
+  if (gifs?.length) {
```

Comment:
What do you think about this ?
```suggestion
  if (gifs?.length) {
    return PostCategory.Picture;
  }
  
  if (files?.length) {
    switch (files[0].fileType) {
      case "image":
        return PostCategory.Picture;
      case "audio":
        return PostCategory.Audio;
      default:
        return PostCategory.VideoNote;
    }
  }
  
  if (title) {
    return PostCategory.Article;
  }
  
  if (message.startsWith("/question")) {
    return PostCategory.Question;
  }
  
  if (message.startsWith("/generate")) {
    return PostCategory.BriefForStableDiffusion;
  }
  
  return PostCategory.Normal;
```

Reply:
It's out of scope since this PR just adds a condition in `getPostCategory` and doesn't refacto other conditions.
But it's better like this !